src/test/java/org/swisspush/redisques/RedisQuesTest.java

```diff
@@ -221,6 +221,7 @@ public void getQueues(TestContext context) {
                 context.assertEquals(OK, message.result().body().getString(STATUS));
                 asyncEnqueue.countDown();
             });
+            Thread.sleep(10);
```

Comment:
Why is this necessary? The test will wait until the countDown has gone from 100 to 0

Reply:
We exhaust the connection pool size without

io.vertx.core.http.ConnectionPoolTooBusyException: Connection pool reached max wait queue size of 24

Maybe we can increase it and then we can remove this again?

Reply:
Is this a general new behaviour of vert.x 4? Or even a general new problem which we have to care about in general? 
Sending 

Maybe sending one after the other (in the event handler) might help?

Reply:
I think we should make the connection pool size configurable and figure out what the limit was before (and then maybe use this limit).

Reply:
The commands went through a queue in the past. So we need to keep in mind that the implementation has changed in this regard (we can now exhaust the max number of connections whereas in the past commands where queued)